### PR TITLE
Add New 'About' View to the App

### PR DESCRIPTION
**Pull request by roundhill**
The iOS app is the only Simplenote app that doesn't have an about screen yet. I've mirrored how the about screen looks on other apps while trying to preserve a more iOS feel:

simulator screen shot - ipad air 2 - 2018-07-11 at 14 31 18
simulator screen shot - iphone 7 - 2018-07-11 at 14 32 01

**To Test**
* Start the app, navigate to Settings -> About.
* The view should display well on all devices, sizes and rotations.
* All of the links open in Safari when tapped, including `Privacy` and `Terms` links which I used an attributed string for.
* Tapping `Done` should close the view.

### CODE REVIEW

File: Simplenote/Classes/SPAboutViewController.swift [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+            break;
+        default:
+            break;
+        }
+        
+        tableView.deselectRow(at: indexPath, animated: false)
+    }
+    
+    @objc func onDoneTap(_ sender:UIButton) {
+        self.dismiss(animated: true)
```
Comment (frosty) [inline Simplenote/Classes/SPAboutViewController.swift]: Again, can remove reference to `self`.

File: Simplenote/Classes/SPAboutViewController.swift [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    private let tableView = UITableView()
+    private let doneButton = UIButton(type: UIButtonType.custom)
+    
+    private let simpleBlue = UIColor(red: 74/255, green: 149/255, blue: 213/255, alpha: 1.0)
+    private let lightBlue = UIColor(red: 118/255, green: 175/255, blue: 223/255, alpha: 1.0)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.view.backgroundColor = simpleBlue
```
[the remark this answers is not in the document]
Comment (roundhill) [reply Simplenote/Classes/SPAboutViewController.swift]: Old habits die hard :)

**Comment by roundhill** [comment]
Thanks for the patch @frosty! Much cleaner than all of that messy stuff in `viewDidLayoutSubviews()`

I also removed the remaining usages of `self` that weren't necessary.

**Comment by roundhill** [comment]
Thank you @frosty!

**Pull request merged**